src/AutoRest.CSharp.V3/Generation/Writers/ClientWriter.cs

```diff
@@ -563,6 +566,8 @@ ref valueVariable
                         case { }:
                             writer.Append($"return {typeof(Response)}.FromValue({valueVariable}, message.Response);");
                             break;
+                        case null when !statusCodes.Any():
+                            break;
```

Comment:
This allows operations to be created that have no responses.

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.empty?view=netframework-4.8

Reply:
I'm not creating an empty enumerable here. There is a null case already (that handles the generic `return message.Response;`) and this new case is specifically for when we have no responseBody and there aren't any statusCodes. This only happens when there aren't any responses (which means every response from that operation will fall into `default` and throw).

Reply:
`!statusCodes.Any() == statusCodes.Empty()` and .Empty is easier to read.

Reply:
As discussed, `.Empty()` is a static method on `Enumerable<T>` and returns an empty enumerable of T.